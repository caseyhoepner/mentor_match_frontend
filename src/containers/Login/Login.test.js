import React from 'react';
import { shallow } from 'enzyme';
import { Login } from './';

describe('Login', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(<Login />)
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});