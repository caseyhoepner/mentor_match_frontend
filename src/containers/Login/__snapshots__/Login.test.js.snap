// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Login />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Mentor Login
        </h1>,
        <form
          className="l-form"
        >
          <input
            className="l-input l-form-item"
            name="email"
            onChange={[Function]}
            placeholder="Enter your email address, mthfkr."
            type="text"
            value=""
          />
          <input
            className="l-input l-form-item"
            name="password"
            onChange={[Function]}
            placeholder="Enter your password, mthfkr."
            type="password"
            value=""
          />
          <button
            className="l-login-btn l-form-item"
          >
            Login
          </button>
        </form>,
      ],
      "className": "l-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Mentor Login",
        },
        "ref": null,
        "rendered": "Mentor Login",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <input
              className="l-input l-form-item"
              name="email"
              onChange={[Function]}
              placeholder="Enter your email address, mthfkr."
              type="text"
              value=""
            />,
            <input
              className="l-input l-form-item"
              name="password"
              onChange={[Function]}
              placeholder="Enter your password, mthfkr."
              type="password"
              value=""
            />,
            <button
              className="l-login-btn l-form-item"
            >
              Login
            </button>,
          ],
          "className": "l-form",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "l-input l-form-item",
              "name": "email",
              "onChange": [Function],
              "placeholder": "Enter your email address, mthfkr.",
              "type": "text",
              "value": "",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "l-input l-form-item",
              "name": "password",
              "onChange": [Function],
              "placeholder": "Enter your password, mthfkr.",
              "type": "password",
              "value": "",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Login",
              "className": "l-login-btn l-form-item",
            },
            "ref": null,
            "rendered": "Login",
            "type": "button",
          },
        ],
        "type": "form",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Mentor Login
          </h1>,
          <form
            className="l-form"
          >
            <input
              className="l-input l-form-item"
              name="email"
              onChange={[Function]}
              placeholder="Enter your email address, mthfkr."
              type="text"
              value=""
            />
            <input
              className="l-input l-form-item"
              name="password"
              onChange={[Function]}
              placeholder="Enter your password, mthfkr."
              type="password"
              value=""
            />
            <button
              className="l-login-btn l-form-item"
            >
              Login
            </button>
          </form>,
        ],
        "className": "l-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Mentor Login",
          },
          "ref": null,
          "rendered": "Mentor Login",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <input
                className="l-input l-form-item"
                name="email"
                onChange={[Function]}
                placeholder="Enter your email address, mthfkr."
                type="text"
                value=""
              />,
              <input
                className="l-input l-form-item"
                name="password"
                onChange={[Function]}
                placeholder="Enter your password, mthfkr."
                type="password"
                value=""
              />,
              <button
                className="l-login-btn l-form-item"
              >
                Login
              </button>,
            ],
            "className": "l-form",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "l-input l-form-item",
                "name": "email",
                "onChange": [Function],
                "placeholder": "Enter your email address, mthfkr.",
                "type": "text",
                "value": "",
              },
              "ref": null,
              "rendered": null,
              "type": "input",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "l-input l-form-item",
                "name": "password",
                "onChange": [Function],
                "placeholder": "Enter your password, mthfkr.",
                "type": "password",
                "value": "",
              },
              "ref": null,
              "rendered": null,
              "type": "input",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Login",
                "className": "l-login-btn l-form-item",
              },
              "ref": null,
              "rendered": "Login",
              "type": "button",
            },
          ],
          "type": "form",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
